x-pack/test_serverless/shared/config.base.ts

```diff
@@ -52,31 +52,31 @@ export default async () => {
       files: [idpPath, jwksPath],
       serverArgs: [
         'xpack.security.authc.realms.file.file1.order=-100',
+        'xpack.security.authc.token.enabled=true',
```

Comment:
__nit:__ as mentioned in Slack, Token Service is enabled automatically when we enable HTTP TLS, so we don't need this. Sorry for the confusion.
```suggestion
```